sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.1.0-preview.1 (Unreleased)
+## 1.0.1
```

Comment:
Also need to add a date "2020-07-07" on this line or the verify change log script will fail at release.